src/pixelflow/core.py

```diff
@@ -208,12 +208,26 @@ def pixelflow(
         else:
             spacing = (1,) * mask.ndim
 
+    if any(val == 0 for val in spacing):
+        raise ValueError("Spacing cannot be zero")
```

Comment:
Might be good to return the spacing in the error message to help with debugging?

```suggestion
        raise ValueError(f"Spacing cannot be zero: {spacing}")
```